internal/service/scaffold/scaffold.go

```diff
@@ -100,7 +100,7 @@ func (s *Service) Run(opts Options) error {
 			contentprovider.ArgDefaultCRFile:      defaultCRFilePath,
 			contentprovider.ArgSecurityConfigFile: securityConfigFilePath,
 		}); err != nil {
-		return fmt.Errorf("%w %s: %w", ErrGeneratingFile, opts.ModuleConfigFileName, err)
+		return fmt.Errorf("%s %w: %w", opts.ModuleConfigFileName, ErrGeneratingFile, err)
```

Comment:
we need to rewrite the ErrGeneratingFile,  ErrCheckingFileExistent and ErrOverwritingFile to match the new error. This way the error is "filename error generating file: .."
so we either change the values of these 3 errors, or rewrite the errors that reference them in a clearer way